BaseTools/Source/Python/ProductDataFormatter/ProductDataFormatter.py

```diff
@@ -104,16 +105,16 @@ def remove_intermediate_files(self):
 
     parser = argparse.ArgumentParser(
         description=(
-            'Formats intermediate OEM data files into '
+            'Formats intermediate product data files into '
             'a single consolidated binary.'))
     parser.add_argument(
         'input_directory',
-        help='Directory of .oemdatabin.i files')
+        help='Directory of .productdatabin.i files')
     parser.add_argument(
         'output_file_path',
-        help='Name of of the final .oemdatabin file.')
+        help='Name of of the final .productdatabin file.')
     args = parser.parse_args()
 
-    oem_data = OemDataFormatter(args.input_directory)
-    oem_data.write_data(args.output_file_path)
-    oem_data.remove_intermediate_files()
+    product_data = ProductDataFormatter(args.input_directory)

```

Comment:
This doesn't appear to be useful history, we can squash it into the previous commit and just introduce the product data formatter without the intermediate oemdata formatter

Reply:
I merged the two commits.